### ISSUE TITLE
License not recognized in pkg.go.dev 

### ISSUE THREAD

**Issue by elafontaine** [google/licensecheck#56]
Hi, 

I'm opening this issue has the license I use for my repository is the MIT License type which I datafilled with Github (I believe).  Github recognize the license and doesn't have any trouble, but the license isn't recognized in the package in pkg.go.dev.

The github repository ; https://github.com/elafontaine/smpp/
The file ; https://github.com/elafontaine/smpp/blob/master/LICENSE
The package link on pkg.go.dev ; https://pkg.go.dev/github.com/elafontaine/smpp

I believe the bug comes from the fact that the license start with : "MIT License"

I compared it with this repository ; https://github.com/fiorix/go-smpp
The license used in that repository ; https://github.com/fiorix/go-smpp/blob/master/LICENSE

**Comment by elafontaine** [comment]
Sorry, I wrote this in a hurry.  

So basically, the check seems to be on the exact text with a couple of templates (like the year and name).  But nothing for the top or bottom of the file.  My instinct tells me there might be a reason for this, but I would like to be sure as I'm pretty sure I generated this from a tool...  I would have to find it back though... if it's important, let me know and I'll search how I did it (been a while).

Otherwise, if this is the desired behavior, I can change my license no problem.
